references/search.md

```diff
@@ -18,7 +18,7 @@ Search for documents matching a specific query in the given index.
 
 | Query Parameter                                                                                   | Description                                                                                     | Default Value |
 | ------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------- | :-----------: |
-| **[q](/guides/advanced_guides/search_parameters.md#query-q)**                                     | Query string \_(mandatory)                                                                      |               |
+| **[q](/guides/advanced_guides/search_parameters.md#query-q)**                                     | Query strings                                                                                   |               |
```

Comment:
Here `q` accepts `strings`. Do you imply that it accepts an array of strings?
It is also not the same description as provided in `guides/advanced_guides/search_parameters.md `

Reply:
```suggestion
| **[q](/guides/advanced_guides/search_parameters.md#query-q)**                                     | Query string                                                                                   |               |
```

oops no, sorry